client/components/suggestion-search/index.jsx

```diff
@@ -81,10 +80,18 @@ class SuggestionSearch extends Component {
 		this.suggestionsRef.handleKeyEvent( event );
 	};
 
-	handleSuggestionMouseDown = position => {
-		this.updateInputValue( position.label );
+	handleSuggestionMouseDown = ( suggestion, suggestionIndex ) => {
```

Comment:
`suggestionIndex` was added to the callback, and `position` was renamed to `suggestion`. It was never a position but a suggestion object like `{ label: 'My Label', value: 'v' }`.